src/profile/profile.ts

```diff
@@ -478,6 +491,22 @@ export class Profile extends Logger {
     );
   }
 
+
+  /**
+   * Gets the identity list.
+   */
+  public async getIdentityAccessList(): Promise<any> {
+    await this.getAddressBook();
```

Comment:
Why is this getter called without using it?
```
public async getAddressBook(): Promise<any> {
    if (!this.trees[this.treeLabels.addressBook]) {
      await this.loadForAccount(this.treeLabels.addressBook);
    }
    return this.trees[this.treeLabels.addressBook];
  }
```
The next line also seems to duplicate the logic?